io.github.mmarco94.tambourine.yml

```
@@ -0,0 +1,60 @@
+app-id: io.github.mmarco94.tambourine
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+add-extensions:
+  org.freedesktop.Platform.ffmpeg-full:
+    directory: lib/ffmpeg
+    version: '22.08'
+    add-ld-path: .
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk17
+command: run.sh
+finish-args:
+  - --filesystem=xdg-run/gvfs
+  - --filesystem=xdg-run/gvfsd
+  # Workaround until https://github.com/LWJGL/lwjgl3/issues/893 is fixed
+  - --filesystem=host:ro
+  - --socket=wayland
+  - --socket=fallback-x11
```

```suggestion
  - --socket=x11
```

It doesn't work on Wayland.

---

Done. I've added https://github.com/MMarco94/tambourine-music-player/issues/3 to investigate why it doesn't work